utils/net.py

```diff
@@ -133,3 +134,22 @@ def resolve_ips(host_iterable, force_dns=False):
             if ip is not None:
                 result.add(ip)
     return result
+
+
+def is_pingable(ip_addr):
+    """verifies the specified ip_address is reachable or not.
+
+    Args:
+        ip_addr: ip_address to verify the PING.
+    returns: return True is ip_address is pinging else returns False.
+    """
+    try:
+        status = os.system("ping -c1 -w2 {}".format(ip_addr))
+        if status == 0:
+            print('IP: {} is UP !'.format(ip_addr))
+            return True
+        print('IP: {} is DOWN !'.format(ip_addr))
+        return False
+    except Exception as e:
+        print(e)
+        return False
```

Comment:
@psav @amavinag I would say that if it is not inside the script, it should only output into a logger. All three messages, first two should use logger.info, the exceptions shoulddo logger.exception(e).